app/controllers/ReadProtectionsController.scala

```diff
@@ -102,6 +103,19 @@ class ReadProtectionsController @Inject() (
       .map(model => sessionCacheService.saveFormData[ProtectionModel]("openProtection", model).map(_ => true))
       .getOrElse(Future.successful(true))
 
+   def saveDormantFixedProtection2016(model: TransformedReadResponseModel)(
+    implicit request: Request[AnyContent]
+  ):  Future[Boolean] =
+    getDormantFixedProtection2016(model)
+      .map(model =>sessionCacheService.saveFormData[ProtectionModel]("fixedProtection2016", model).map(_ => true))
```

Comment:
Could you use `saveProtection` method instead? 

Mapping to `true` also makes no sense, as the returned value is not used anywhere. You can simply return `Unit` in this case.